h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -103,39 +109,14 @@
       });
     }
 
-    function injectIntoLocalDocument(tab) {
-      if (isPDFURL(tab.url)) {
-        return injectIntoLocalPDF(tab);
-      } else {
-        return Promise.reject(new h.LocalFileError('Local non-PDF files are not supported'));
-      }
-    }
-
-    function injectIntoRemoteDocument(tab) {
-      return isPDFURL(tab.url) ? injectIntoPDF(tab) : injectIntoHTML(tab);
-    }
-
-    function injectIntoPDF(tab) {
```

Comment:
We don't. ^_^ We simply load the viewer (see `pdfHandler.js`) in which we're already embedded.